app/controllers/concerns/with_organization.rb

```diff
@@ -8,6 +8,6 @@ def visit_organization!
   end
 
   def implicit_central?
-    !(request.first_subdomain == 'central') && Organization.central?
+    request.empty_subdomain? && Organization.central?
```

Comment:
If subdomain is empty Organization will always be central, right? Has it sense to check bot things?